tests/ui/version/version-info-flags.rs

```diff
@@ -0,0 +1,9 @@
+// Check that rustc accepts various version info flags.
+//@ dont-check-compiler-stdout
+//@ revisions: version verbose-version long-verbose-version
+//@[version] check-pass
```

Comment:
This check-pass needs to be unconditional

```
//@ check-pass
```

Reply:
Right, the version I had managed to restore from the last PR still had the check-pass redundancy. Thanks for the fix!